### ISSUE TITLE
build error "both ‘const’ and ‘constexpr’ cannot be used here" in 4a255351

### ISSUE THREAD

**Issue by larkost** [rethinkdb/rethinkdb#3533]
In `next` 5111ff2 I am getting a build error on newton (so GCC) but not on MacOS:

```
src/rdb_protocol/terms/db_table.cc:376:61: error: both ‘const’ and ‘constexpr’ cannot be used here
src/rdb_protocol/terms/db_table.cc:377:58: error: both ‘const’ and ‘constexpr’ cannot be used here
src/rdb_protocol/terms/db_table.cc:378:59: error: both ‘const’ and ‘constexpr’ cannot be used here
src/rdb_protocol/terms/db_table.cc:379:56: error: both ‘const’ and ‘constexpr’ cannot be used here
```

`git blame` points to 4a255351 as the likely suspect so assigning to @Tryneus

**Comment by Tryneus** [comment]
Fixed in `next` in commit a813df7, OTS review by @vexocide, will be in release 1.16.

**Comment by VeXocide** [comment]
For future reference, `static constexpr char const * const foo = "foo";` is technically correct but not supported by GCC 4.6.
